README.md

```diff
@@ -343,6 +343,7 @@ Additional lists you might find useful:
 ## Security
 *Plugins and information around security, preventing vulnerabilities and protection against XSS and alike.*
 
+- :strawberry: [Ali1/BruteForceProtectionComponent](https://github.com/Ali1/cakephp-brute-force-protection/) - Simple way to add Brute Force Protection to your installation without involving database.
```

Comment:
Doesnt look like this is the actual plugin name. See how to set the proper name as per guidelines.